networking/src/Models/AWS/AWSModelGameQuestionConnection.ts

```diff
@@ -0,0 +1,19 @@
+import { AWSQuestionTemplate } from "./AWSQuestionTemplate";
+import { AWSGameTemplate } from "./AWSGameTemplate";
+
+export type AWSModelGameQuestionConnection = {
+  items: Array<AWSGameQuestions>;
+  __typename: string;
```

Comment:
Same as other one, I don't think we need `__typename` here, and in any other models on client side

Reply:
fixed